src/units/abilities.cpp

```diff
@@ -1256,6 +1256,38 @@ namespace { // Helpers for attack_type::special_active()
 
 }//anonymous namespace
 
+unit_ability_list attack_type::get_specials_and_abilities_with_priority(const std::string& special) const
+{
+	//the custom_priority attribute is used for define rules of priority of damage type
```

Comment:
This comment mentions damage type, but the damage type case no longer exists.

Reply:
corrected